### PR TITLE
Storage gui keep mountpoint elements expanded

### PR DESCRIPTION
**Pull request by rvykydal**

### CODE REVIEW

**Comment by rvykydal** [comment]
There are update images adding the first one or both patches to F32 Beta (works with today's latest F32 as well) to check the new behavior. The links are in the BZ https://bugzilla.redhat.com/show_bug.cgi?id=1210944

**Comment by rvykydal** [comment]
jenkins, test this please

**Comment by rvykydal** [comment]
@poncovka 
In https://github.com/rhinstaller/anaconda/pull/2404/commits/d4e2edd77ef118f721e8d3ca42eb9c0293299428 I moved the functionality to Accordion (replaced expand_page with expand_pages).
As for the  init_expanded_pages this belongs to the spoke IMO, not in the Accordion (we need first_page in the spoke anyway). Also it is not just expand_first_page, but also do not keep pages expanded from the refresh() parameter point of view so I keep the name.
I'll squash the commits before merge when/if ACKed.

File: pyanaconda/ui/gui/spokes/lib/accordion.py [hunk 1]
```diff
@@ -279,14 +279,20 @@ def page_for_selector(self, selector):
...
 
-    def expand_page(self, page_title):
-        page = self.find_page_by_title(page_title)
-        expander = page.get_parent()
-        if not expander:
-            raise LookupError()
-
-        if not expander.get_expanded():
-            expander.emit("activate")
+    def get_expaned_pages(self):
```
Comment (poncovka) [inline pyanaconda/ui/gui/spokes/lib/accordion.py]: `expanded`?

Comment (rvykydal) [reply pyanaconda/ui/gui/spokes/lib/accordion.py]: Fixed

**Comment by rvykydal** [comment]
I've just squashed the three ACKed commits into a single one.

**Comment by rvykydal** [comment]
jenkins, test this please

**Comment by rvykydal** [comment]
Closing, I've created new PR based on f32-devel: https://github.com/rhinstaller/anaconda/pull/2409

**Pull request closed without merging**